lib/src/navbar_router.dart

```diff
@@ -397,6 +463,96 @@ class _NavbarRouterState extends State<NavbarRouter>
                         },
                         menuItems: items),
                   ),
+
+                  // swipe area
+                  Positioned.fromRect(
+                    rect: !widget.swipeable
+                        ? Rect.zero
+                        : widget.swipeableArea ??
+                            Rect.fromLTWH(
+                                getPadding(),
+                                kDragAreaTop,
+                                kDragAreaWidth,
+                                MediaQuery.of(context).size.width *
+                                    kDragAreaHeightFactor),
+                    child: GestureDetector(
+                      key: const ObjectKey("swipe-left"),
+                      behavior: HitTestBehavior.translucent,
+                      onHorizontalDragStart: (details) {
```

Comment:
create separate methods out of the build method `handleDragStart`,`handleDragUpdate`, and `handleDragEnd` and move this logic there and move as much logic as possible in the corresponding methods of Gesture class to keep the build method clean and readable